sdk/identity/Azure.Identity/tests/SessionRecords/ClientCertificateCredentialLiveTests/FromCertificatePath(False).json

```diff
@@ -109,7 +109,8 @@
         "x-client-Ver": "4.30.1.0",
         "x-ms-client-request-id": "1d5ac3e8db5dd0962fa4321bd098365a",
         "x-ms-lib-capability": "retry-after, h429",
-        "x-ms-return-client-request-id": "true"
+        "x-ms-return-client-request-id": "true",
+        "cookie": "x-ms-gateway-slice=estsfd; stsservicecookie=estsfd"
```

Comment:
These cookies are now present in requests because the recording happens later in the pipeline than with the legacy framework.

Reply:
Since we are planning to disable cookies, I will add "cookie" to the ignored headers for now.